src/app/lib/config/services/punjabi.js

```diff
@@ -169,6 +169,10 @@ export const service = {
         title: 'ਕੌਮਾਂਤਰੀ',
         url: '/punjabi/international',
       },
+      {
+        title: '#ISWOTY',
+        url: '/gujarati/resources/idt-0370808c-2dea-4f55-b69f-309a2881ef25',
```

Comment:
weird that this is a link to Gujarati. I can see this link on the old front page: `/punjabi/resources/idt-681afed0-03f5-4d5f-a640-3011a722d1c8`. Could you double check with Jon?

Reply:
Good catch 👍Double checked with Jon and updated the link